google-basemap/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "carto-deckgl-example-basemaps",
```

Comment:
Rename to carto-deckgl-example-google-maps

Reply:
Done